server/shared/src/main/scala/org/http4s/server/staticcontent/ResourceService.scala

```diff
@@ -82,13 +82,13 @@ class ResourceServiceBuilder[F[_]] private (
 
   def withBufferSize(bufferSize: Int): ResourceServiceBuilder[F] = copy(bufferSize = bufferSize)
 
-  def toRoutes(implicit F: Async[F]): HttpRoutes[F] = {
+  def toRoutes(implicit F: Async[F]): F[HttpRoutes[F]] = {
```

Comment:
Another signature change here, that caused the test suite to need major refactoring. We could, also, just not log ...